db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/task/schedule/CronSchedule.java

```diff
@@ -88,15 +97,32 @@ private void lazyInitExecutionTime() {
         if (isDisabled()) {
           cronExecutionTime = new CronSchedule.DisabledScheduleExecutionTime();
         } else {
-          CronParser parser =
-              new CronParser(CronDefinitionBuilder.instanceDefinitionFor(CronType.SPRING53));
+          CronParser parser = new CronParser(CronDefinitionBuilder.instanceDefinitionFor(cronType));
```

Comment:
Can we do the translation here instead? (for `cronType`) 
(Also handlings potential nulls)